client/galaxy/scripts/onload.js

```diff
@@ -185,7 +185,7 @@ $(document).ready( function() {
     }
 
     // Load all webhooks with the type 'onload'
-    $.getJSON('/api/webhooks/onload/all', function(webhooks) {
+    $.getJSON( Galaxy.root + '/api/webhooks/onload/all', function(webhooks) {
```

Comment:
Should be 'api...' instead of '/api...' right?

Reply:
Both actually work, but I'd rolled that fix into my util function refactoring of this.  If you want, I'll PR that change here for consistency.

Reply:
(did PR'd at #4715; might as well be consistent even with it being rewritten in -dev I guess)